db/schema.rb

```diff
@@ -23,12 +23,12 @@
     t.string "description"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "status"
   end
 
   create_table "pet_applications", force: :cascade do |t|
     t.bigint "application_id", null: false
-    t.bigint "pet_id", null: false
-    t.string "status"
+    t.bigint "pet_id"
```

Comment:
Same as previous comment - may want to look back at not allowing nulls in the future.